docs/source/05_data/01_data_catalog.md

```diff
@@ -589,3 +590,34 @@ io.save("ranked", ranked)
 ```
 
 > *Note:* Saving `None` to a dataset is not allowed!
+
+# Using Minio (S3 API Compatible Storage) in Data Catalog
+Minio has a S3 Compatible API, if you have security concern, you may consider hosting the storage within your network.
+
+The easiest way to setup Minio is to run a docker.
```

Comment:
```suggestion
The easiest way to setup MinIO is to run a Docker image.
```

Reply:
I updated the docs again, I think I removed these paragraph in earlier commit already, not sure why it here again. Everything should be under Example 14: section

Reply:
I think the maintainer can just add commits too, feel free to directly edit it.